nixos/modules/flyingcircus/services/sensu/client.nix

```diff
@@ -251,6 +251,13 @@ in {
         '';
         interval = 600;
       };
+      security = {
+        notification = "Vulnerabilities found.";
+        command = ''
+          ${pkgs.vulnix}/bin/vulnix -m http://downloads.fcio.net/nvd/ --system
```

Comment:
It is not clear what keeps our mirror current and if we have monitoring for that (and I don't understand why we need this mirror at all).